tasks/managed/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -122,7 +122,8 @@ spec:
           else
               COSIGN_REKOR_ARGS="--insecure-ignore-tlog=true"
           fi
-          verify_output=$(run_cosign verify "$COSIGN_REKOR_ARGS" "$reference")
+          # shellcheck disable=SC2086
```

Comment:
Why is this needed? AFAICS, it's really just one argument, so `"$COSIGN_REKOR_ARGS"` should work fine, no? If you needed to use more arguments, then I would suggest to use an array and then expand the arguments using `"${COSIGN_REKOR_ARGS[@]}"`, see e.g. here https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/managed/create-pyxis-image/create-pyxis-image.yaml#L165

Reply:
fixed